source/css/style.css

```diff
@@ -2,77 +2,546 @@
+html {
+    font-size: 10px;
+    height: calc(100%);
 }
-#play_field table {
+body {
     width: 100%;
     height: 100%;
+    background-color: rgb(240, 240, 240);
+    font-family: "Myriad Pro", Calibri, sans-serif;
+    line-height: 1;
 }
-#play_field td {
-    max-width: 30%;
-    min-width: 50px;
-    max-height: 30%;
-    min-height: 50px;
-    text-align: center;
-    vertical-align: middle;
-    font-size: 3em;
-    line-height: 0em;
-    cursor: pointer;
+ol, ul {
+    list-style: none;
 }
-#start_button, #start_game {
-    margin: 10px 0;
+.full_width {
+    width: 100%;
 }
-.cell.cross{}
-.cell.nought{}
-.status {
+.container {
+    max-width: 70vmin;
+}
+.centered {
+    margin-right: auto;
+    margin-left: auto;
+}
+.wrapper {
+    position: relative;
     display: block;
-    font-size: 30px;
+    margin: auto;
+    resize: both;
+}
+.transformer {
+    position: relative;
+    width: 300rem;
+    -webkit-transform-origin: top left;
+    -moz-transform-origin: top left;
+    -ms-transform-origin: top left;
+    -o-transform-origin: top left;
+    transform-origin: top left;
+}
+.collapsed {
+    display: none !important;
+}
+button:hover {
+    cursor: pointer;
 }
-.winner {
-    background-color: green;
+button:focus {
+    outline: none !important;
 }
-.form-inline .form-control.cell_count {
-    width: 50px;
+
+/*** Flex ***/
+.flex_column {
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.flex_column_item {
+    flex-basis: auto;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+.flex_row {
+    display: -webkit-flex; /* Safari */
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    height: auto;
+}
+.flex_row_item {
+    flex-basis: auto;
+    flex-grow: 1;
+    flex-shrink: 1;
+    margin: auto;
+}
+
+
+/*** HEADER ***/
+header, footer {
+    position: relative;
+    background-color: rgb(52, 52, 52);
+}
+header {
+    height: 10vh;
+    -webkit-box-shadow: 0 1vh 6vh rgba(0,0,0,0.5);
+    -moz-box-shadow: 0 1vh 6vh rgba(0,0,0,0.5);
+    box-shadow: 0 1vh 6vh rgba(0,0,0,0.5);
+}
+header .wrapper {
+    height: 10vh;
+}
+header .transformer {
+    top: calc(100% / 7);
+
+}
+header .header_block {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+}
+.game_title {
+    order: 2;
+    flex-grow: 3;
+    width: 240rem;
+    color: rgb(81, 168, 255);
+}
+.game_title h2 {
+    margin: 0;
+    font-size: 30rem;
+    font-weight: bold;
     text-align: center;
-    margin: 0 auto;
+}
+#start_game {
+    order: 1;
+    flex-grow: 1;
+    width: 30rem;
+}
+#settings_btn {
+    order: 3;
+    flex-grow: 1;
+    width: 30rem;
+    text-align: right;
+}
+#start_game > button,
+#settings_btn > button {
+    width: 30rem;
+    height: 30rem;
+    padding: 0;
+    border: none;
+    -webkit-border-radius: 50%;
```

Comment:
We've agreed to support only newest browser versions. Do we still need these specific styles despite this fact?